src/Stripe.net/Entities/StripeProduct.cs

```diff
@@ -81,14 +81,26 @@ public class StripeProduct : StripeEntityWithId
         /// Whether this product is a shipped good.
         /// </summary>
         [JsonProperty("shippable")]
-        public bool Shippable { get; set; }
+        public bool? Shippable { get; set; }
 
         /// <summary>
         /// A sublist of active SKUs associated with this product.
         /// </summary>
         [JsonProperty("skus")]
         public StripeList<StripeSku> Skus { get; set; }
 
+        /// <summary>
+        /// What appears on a customers credit card statement. Max of 22 characters.
```

Comment:
customer's
I'd also remove the 22 chars mention, better to keep this in the official docs.